frontend/packages/console-shared/src/components/badges/DevPreviewBadge.tsx

```diff
@@ -1,8 +1,14 @@
 import * as React from 'react';
 import { Label } from '@patternfly/react-core';
+import './Badge.scss';
```

Comment:
Odd - how did this work before?

Reply:
It would directly using the `tech-preview` class, not sure if it would work without the import.